src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -152,35 +111,74 @@
  * Example:
  * </p>
  * <pre>
- * switch (i) {
- *   case 1:
- *     break;
- *   case 2: // Previous case: OK
- *           // This case: violation, last case must have statement that transfer control
- *     i++;
+ * public class Example2 {
+ *   public void foo() throws Exception {
+ *     int i = 0;
+ *     while(i &gt;= 0) {
+ *       switch(i) {
+ *         case 1:
+ *           i++;
+ *         case 2: // violation
+ *           i++;
+ *           break;
+ *         case 3:
+ *           i++;
+ *           return;
+ *         case 4:
+ *           i++;
+ *           throw new Exception();
+ *         case 5:
+ *           i++;
+ *         case 6: // violation
+ *         case 7:
+ *           i++;
+ *           continue;
+ *         case 11: // violation
+ *           i++;
+ *       }
+ *     }
+ *   }
  * }
  * </pre>
  * <p>
  * To configure the check with custom relief pattern:
  * </p>
  * <pre>
  * &lt;module name=&quot;FallThrough&quot;&gt;
- *    &lt;property name=&quot;reliefPattern&quot; value=&quot;FALL?[ -]?THROUGH&quot;/&gt;
+ *    &lt;property name=&quot;reliefPattern&quot; value=&quot;no break by design&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
  * Example:
  * </p>
  * <pre>
- * switch (i) {
- *   case 1:
- *     i++;
- *     // FALL-THROUGH
- *   case 2: // OK, "FALL-THROUGH" matches the regular expression "FALL?[ -]?THROUGH"
- *     i++;
- *     // fall-through
- *   case 3: // violation, "fall-through" doesn't match
- *     break;
+ * public class Example3 {
+ *   public void foo() throws Exception {
+ *     int i = 0;
+ *     while(i &gt;= 0) {
+ *       switch(i) {
+ *         case 1:
+ *             i++;
+ *         case 2: // violation
+ *           i++;
+ *           break;
+ *         case 3:
+ *           i++;
+ *           return;
+ *         case 4:
+ *           i++;
+ *           throw new Exception();
+ *         case 5:
+ *           i++; // no break by design
```

Comment:
this line should be same in all examples. keep comment in all examples.

Reply:
ok. will do it.